app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,6 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.all
+    @sorted_ingredients = Ingredient.sorts_a_z
```

Comment:
Ext 1 essentially overrides US 1 in which case it would've been acceptable to get rid of line 3 altogether. Especially because you are no longer using `@ingredients` in your view. That means line 3 is making a call to our database to get ALL of our ingredients and then do nothing with them. That gets pretty expensive for data that we don't need.